From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect struct ath10k_hw_params
Patch-mainline: never, kabi
References: kabi

In 4.4.132, commit b88fb9ea475a78a627ff83ddbb1fcbac3f605187 (ath10k: fix
rfc1042 header retrieval in QCA4019 with eth decap mode), upstream
commit 2f38c3c01de945234d23dd163e3528ccb413066d added decap_align_bytes
to struct ath10k_hw_params. It made the kABI checker to complain.

Given it is ath10k-internal structure only, just hide it from the kABI
checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/net/wireless/ath/ath10k/core.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/wireless/ath/ath10k/core.h
+++ b/drivers/net/wireless/ath/ath10k/core.h
@@ -671,8 +671,10 @@ struct ath10k {
 			size_t board_ext_size;
 		} fw;
 
+#ifndef __GENKSYMS__
 		/* Number of bytes used for alignment in rx_hdr_status */
 		int decap_align_bytes;
+#endif
 
 	} hw_params;
 
